src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const declarations = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. The `split` method doesn't mutate the original string, so you can chain methods directly after it without the need for creating an intermediate `declarations` array.